src/cpp/rediscluster.cpp

```diff
@@ -1075,6 +1075,24 @@ inline CommandReply RedisCluster::_run(const Command& cmd, std::string db_prefix
 // Connect to the cluster at the address and port
 inline void RedisCluster::_connect(SRAddress& db_address)
 {
+    // Build a connections object for this connection
+    // No need to repeat the build on each connection attempt
+    // so we do it outside the loop
+    sw::redis::ConnectionOptions connectOpts;
+    if (db_address._is_tcp) {
+        connectOpts.host = db_address._tcp_host;
+        std::cout << "Connecting to host: " + connectOpts.host + "; ";
```

Comment:
Also here capture to logger, or eliminate the line

Reply:
same